shell/src/main/java/alluxio/cli/fsadmin/report/CapacityCommand.java

```diff
@@ -189,6 +174,84 @@ private void printAggregatedInfo(GetWorkerReportOptions options) {
     }
   }
 
+  /**
+   * Prints worker capacity information.
+   *
+   * @param workerInfoList the worker info list to get info from
+   */
+  private void printWorkerInfo(List<WorkerInfo> workerInfoList) {
+    mIndentationLevel = 0;
+    if (mCapacityTierInfoMap.size() == 0) {
+      return;
+    } else if (mCapacityTierInfoMap.size() == 1) {
+      // Do not print Total value when only one tier exists
+      printShortWorkerInfo(workerInfoList);
+      return;
+    }
+
+    String workerInfoFormat = "%-16s %-16s %-13s %-16s %s";
```

Comment:
better to have util methods to help make the layout of print out --- in this case, it is easy to maintain across product code and test code